alibi/utils/tests/test_distributed.py

```diff
@@ -252,12 +250,13 @@ def test_invert_permutation(permutation_generator):
 
 
 # MockExplainer args and kwargs
-explainer_init_args = [[(0.01,), (0.05, )], ]   # type: ignore
+explainer_init_args = [[(0.01,), (0.05,)], ]  # type: ignore
 explainer_init_kwargs = [[{'multiplier': 2}, {'multiplier': 3}], [{'multiplier': 2}, ]]  # type: ignore
 
 batch_size = [2]
 ncpus = [1, len(explainer_init_kwargs[0])]  # set so that both the error raising inputs and correct inputs are tested
-actor_cpu_fraction = [1.0, 0.5, None]
+# TODO: check that redefining these (type: ignore) actually makes the tests run with the right parameters
+actor_cpu_fraction = [1.0, 0.5, None]  # type: ignore
```

Comment:
All `type: ignore` in this module are due to vairable re-definiton. This brings up a point, how is `pytest` running the correct test cases...?

Reply:
Well that is strange/worrying...